src/System.Private.ServiceModel/tools/scripts/StopWcfSelfHostedSvc.cmd

```diff
@@ -25,10 +38,38 @@ if '%_runelevated%' == '' (
 )
```

Comment:
I understand, but those are from a different PR and I would prefer to lump those into the discussion about moving to an EXE.  In a nutshell, runElevated.vbs was able to run only CMD files and did not behave correctly attempting to do a TASKKILL alone.  Thus, StopWcfSelfHostedProcess was created to that it could be invoked from StopWcfSelfHostedSvc.  I can rename here if that helps, but having umpteen scripts to try to orchestrate what is conceptually simpler as an EXE seems like throw-away work.